tests/test_arrow.py

```diff
@@ -1776,7 +1776,7 @@ def test_multiple_granularity(self):
             self.now.humanize(
                 later108onlydistance, only_distance=True, granularity=["year", "second"]
             )
-            == "3 years and 5327200 seconds"
+            == "3 years and 5392000 seconds"
```

Comment:
Why did this change? Is it because of the 365.25 turning to 365?

Reply:
Yeah man, 3 years of 0.25 is 64800 seconds. I'll add a comment.